### PR TITLE
[16.0][MIG] mrp_subcontracting_partner_management: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

@Tecnativa TT47844

### CODE REVIEW

File: mrp_subcontracting_partner_management/models/res_partner.py [hunk 1]
```diff
@@ -197,22 +184,20 @@ def _create_subcontracting_location_data(self, vals):
 
     def _create_operation_type_for_subcontracting(self, vals):
         # Creating Operation Type for Subcontracting starts here
-        operation_type_rec_id, _ = self._create_subcontracted_operation_type(vals)
-        return {"partner_picking_type_id": operation_type_rec_id}
+        pickint_type_id, _ = self._create_subcontracted_operation_type(vals)
```
Comment (pedrobaeza) [inline mrp_subcontracting_partner_management/models/res_partner.py]:
```suggestion
        picking_type_id, _ = self._create_subcontracted_operation_type(vals)
```

**Review by pedrobaeza — APPROVED** [review]

File: mrp_subcontracting_partner_management/README.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To see newly created rules, go to **Settings** > **Inventory** and activate "Multi step routes".
+
+It is also possible to allow a user to check inventory locations in **Sales & Purchases** tab of **Vendor** without activating dev mode.
+To do so, enable technical extra right **Display Inventory section on Vendor** in user.
```
Comment (chienandalu) [inline mrp_subcontracting_partner_management/README.rst]: Couldn't find this group...

File: mrp_subcontracting_partner_management/README.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Usage
+=====
+
+* Select a partner of type "Company"
+* Enable the "Subcontractor" checkbox
+* New entities are created or existing are used if were created previously
+* When disabled all associated enties will be archived
+* When name of subcontractor is updated, names of entities are updated automatically.
+* It is also possible to check inventory locations using **Subcontractor Location Stock** smart button on partner.
+* When name of subcontractor is updated, names of entities (subcontracting location, operation type, rules) are updated automatically.
```
Comment (chienandalu) [inline mrp_subcontracting_partner_management/README.rst]: This one is duplicated with the point two from lines above

**Review by chienandalu — COMMENTED** [review]
Tested functionally. Just some comments on the readme:

**Review by chienandalu — APPROVED** [review]

**Pull request merged**